libs/payments/ui/src/lib/nestapp/app.ts

```diff
@@ -41,4 +41,11 @@ class AppSingleton {
   }
 }
 
-export const app = singleton('nestApp', new AppSingleton()) as AppSingleton;
+export async function reinitializeNestApp() {
+  return (
+    singleton('nestApp', new AppSingleton(), true) as AppSingleton
+  ).initialize();
+}
+
+export const getApp = () =>
```

Comment:
Exporting a function `getApp` instead of the constant `app`, ensures that Next.js pages always fetches the latest `AppSingleton`